src/main/webapp/view/view.js

```diff
@@ -37,7 +37,10 @@ async function initialize() {
   setLectureName();
   loadVideoApi();
   intializeDiscussion();
-  loadTranscript();
+  // TODO: Move TranscriptArea initialization outside of initialize()
+  // and replace string parameter with a controller object.
+  const transcript = new TranscriptArea('event controller');
+  transcript.loadTranscript();
```

Comment:
Calling the server here, let this be async. As a standard we should not block the UI if a network call is being made.

Reply:
thanks for the info!